pages/api/v1/posts/comments/getCommentByPostId.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { NextApiHandler } from 'next';
+import withErrorHandling from '~src/api-middlewares/withErrorHandling';
+import { isValidNetwork } from '~src/api-utils';
+import { postsByTypeRef } from '~src/api-utils/firestore_refs';
+import messages from '~src/util/messages';
+import { getComments } from '../on-chain-post';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const getPostComments = async ({ postId, network, postType, pageSize, lastDocumentId }: {
+	postId: string, network: string, postType: any, pageSize: any,
```

Comment:
better types here please

Reply:
postType has a type, pageSize should be number etc.

Reply:
Yes, I'll update that